cmake/modules/CUDA.cmake

```diff
@@ -29,9 +29,9 @@ if(USE_CUDA)
     message(FATAL_ERROR "Cannot find CUDA, USE_CUDA=" ${USE_CUDA})
   endif()
   message(STATUS "Build with CUDA ${CUDA_VERSION} support")
-  tvm_file_glob(GLOB RUNTIME_CUDA_SRCS src/runtime/cuda/*.cc)
+  tvm_file_glob(GLOB RUNTIME_CUDA_SRCS ${CMAKE_CURRENT_LIST_DIR}/../../src/runtime/cuda/*.cc)
```

Comment:
I think you could use https://cmake.org/cmake/help/latest/variable/PROJECT_SOURCE_DIR.html to avoid having to go up directories.